force-app/main/default/lwc/githubViewerContainer/githubViewerContainer.js

```diff
@@ -0,0 +1,37 @@
+import { LightningElement } from 'lwc';
+
+export default class GithubViewerContainer extends LightningElement {
+    currentStage = 'listOfCommitsStage';
+    detailsCommit;
+
+    get listOfCommitsStage() {
```

Comment:
[MAJOR/DOUBT]
Trochę niejasne te nazwy. Sugerują zwrotkę listy albo stage a to zwraca booleana